static/app/views/integrationPipeline/githubInstallationSelect.tsx

```diff
@@ -90,9 +105,36 @@ export function GithubInstallationSelect({installation_info}: GithubInstallation
         />
 
         <ButtonContainer>
-          <StyledButton onClick={handleSubmit} disabled={isSaving || !installationID}>
-            Install
-          </StyledButton>
+          {installationID === '-1' || has_scm_multi_org ? (
```

Comment:
i personally find it a bit hard to read ternary operators so it might be helpful to turn this into a function that can render the install / upsell button

i'm also not sure you need to render the upsell with "Feature Not Available" for self-hosted, you could just have a simple disabled button instead?

pseudocode
```
const renderInstallButton = () => {
    if (installationID === '-1' || has_scm_multi_org) {
        return <StyledButton>Install</StyledButton>
    }

    if (isSelfHosted) {
        return <StyleButton disabled={true}>Feature Not Available</StyledButton>
    }
    return <Upsell />
}
```